### ISSUE TITLE
v2.13.0 Stopped working on internal display

### ISSUE THREAD

**Issue by schroef** [emoacht/Monitorian#134]
Noticed today that v2.13.0 stopped working on my internal display. It was working earlier this week. I tried restarting the app couple times, going to single monitor vs dual monitor. Did not solve this.

**Comment by schroef** [comment]
I did a system restart and now works again. I did notice i had a windows update notification, i doubt that has anything to do with it right? Im runnin Windows 10 20H2

**Comment by emoacht** [comment]
Thank you for the report.
The issue has been repeated after the restart?
I have no idea whether there is something to do with the notification.

**Comment by schroef** [comment]
Ive bumped into it only once now. Ill keep an eye out and check system event log next time. Didnt think of that when it happened

**Comment by schroef** [comment]
Ive assigned a shortcut to the app using an alias, used to work fine. But currently app wont show. I see the taskbar appear but not the app. I really dislike how windows is so slowing an app when shortcut is assigned.  The Application Frame Host is really slowing down things. Do you perhaps know of a solution to that? I know its of topic, so perhaps i should make a new post?

**Comment by emoacht** [comment]
@schroef Could you raise another issues by `Reference in new issue`?
Then what you actually do by using an alias.?

**Comment by emoacht** [comment]
Any new development? If no, I am closing this issue.

**Comment by schroef** [comment]
Sorry didnt find my solution yet. ill close this thread and make a new issue
